sql_generators/glean_usage/event_monitoring_live.py

```diff
@@ -69,8 +69,8 @@ def generate_per_app_id(
         artifacts = []
 
         if not self.no_init:
```

Comment:
I'm not sure if these `self.no_init` checks are still applicable.

Reply:
I guess we can change this to `True`: https://github.com/mozilla/bigquery-etl/blob/5c2682a9cea4a8b5c984bfc9c71c771e5ec7f18c/sql_generators/glean_usage/event_error_monitoring.py#L25 and remove the `if not self.no_init`